quality_of_service_interactive_demo/publisher_with_qos/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>publisher_with_qos_demo</name>
+  <version>0.7.3</version>
+  <description>Publisher with configurable QoS policy settings</description>
+  <maintainer email="miaofei@amazon.com">Miaofei Mei</maintainer>
```

Comment:
You should make this `ros-contributions@amazon.com` instead of your individual email - but if you wrap this into the existing package, this file will go away